src/api/data.js

```diff
@@ -44,18 +44,16 @@ function userDetailsChangedChannel() {
 }
 
 function transformBooks(bookData) {
-  if (bookData.isbns.length === 0) {
-    bookData.isbns = [{}];
-  }
-  return { author: bookData.author, title: bookData.title, ISBN: bookData.isbns[0].isbn13 };
+  return { author: bookData.author, title: bookData.title, ISBN: bookData.primary_isbn13, coverImagePath: bookData.book_image };
 }
 
 const hasISBN = book => (book.ISBN);
 
 function getRemoteBooks() {
-  return fetch('https://api.nytimes.com/svc/books/v3/lists/best-sellers/history.json?api-key=8742500acd2b41a9a1143b62c7e557c9')
+  return fetch('https://api.nytimes.com/svc/books/v3/lists/overview.json?api-key=8742500acd2b41a9a1143b62c7e557c9')
   .then(data => data.json())
-  .then(response => response.results.map(transformBooks).filter(hasISBN));
+  .then(json => json.results.lists[10].books.concat(json.results.lists[9].books))
+  .then(response => response.map(transformBooks).filter(hasISBN));
 }
 
```

Comment:
Looks great - the one thing I would suggest here to make this a bit more readable is to convert the `10` and the `9` to `const`'s

eg

```diff
const hasISBN = book => (book.ISBN);

+ const FICTION_BOOKS = 10;
+ const YOUNG_ADULT_HARD_COVER = 9;

function getRemoteBooks() {
  return fetch('https://api.nytimes.com/svc/books/v3/lists/overview.json?api-key=8742500acd2b41a9a1143b62c7e557c9')
  .then(data => data.json())
+ .then(json => json.results.lists[FICTION_BOOKS].books
+    .concat(json.results.lists[YOUNG_ADULT_HARD_COVER].books))
  .then(response => response.map(transformBooks).filter(hasISBN));
}
```